packages/jasmine/jasmine_runner.js

```diff
@@ -154,22 +144,66 @@ async function main(args) {
     }
   }
 
-  await jrunner.execute();
+  // Jasmine versions prior to 3.10.0 should use the old API.
+  if (/^3\.[1-9]\.|^2\./.test(jrunner.coreVersion())) {
+    console.warn(`DEPRECATED: Support for Jasmine versions prior to '3.10.x' is deprecated in '@bazel/jasmine'.`);
```

Comment:
looks like yes, we should have the `TODO(6.0)` to drop it

Reply:
Sure just give me a couple of mins